concrete.mk

```diff
@@ -37,6 +37,7 @@ REBARC = $(REBAR) -C $(REBAR_CONFIG)
 
 # For use on Travis CI, skip dialyzer for R14 and R15. Newer versions
 # have a faster dialyzer that is less likely to cause a build timeout.
+SKIP_DIALYZER ?= false
 DIALYZER = dialyzer
 R14 = $(findstring R14,$(TRAVIS_OTP_RELEASE))
```

Comment:
instead of relying on travis_otp_release can we just ask erl? (I guess that question is more for concrete than for this specific copy...)